diablo/sql_templates/update_rds_sis_sections.template.sql

```diff
@@ -49,10 +49,10 @@ INSERT INTO sis_sections (allowed_units, course_name, course_title, instruction_
     instructor_uid VARCHAR(80),
     is_primary BOOLEAN,
     meeting_days VARCHAR(80),
-    meeting_end_date VARCHAR(80),
+    meeting_end_date TIMESTAMP,
     meeting_end_time VARCHAR(80),
     meeting_location VARCHAR(80),
-    meeting_start_date VARCHAR(80),
+    meeting_start_date TIMESTAMP,
```

Comment:
@paulkerschen - This PR is ready for review. I just added the two lines above. I had success in `diablo-qa` after manually editing SQL template with the change in PR #334.